content/docs/user-guide/components/reference/terrain/terrain_physics_collider.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Terrain Physics Collider component
+linktitle: TerrainPhysicsCollider
+description: ' Open 3D Engine (O3DE) Terrain Physics Collider reference. '
+weight: 100
+---
+
+The **Terrain Physics Collider** component provides terrain data to the physics system in the form of a heightfield and material assignments.  
+You can configure the dimensions of the collider by modifying the [Axis Aligned Box Shape Component](/docs/user-guide/components/reference/shape/axis-aligned-box-shape) on the same entity.
+The **Box component** is required for the **Terrain Physics Collider** to operate.
```

Comment:
```suggestion
```

Suggest moving this to a Dependencies heading to match style of other components.

Reply:
Done